example/lib/screens/user_screen.dart

```diff
@@ -435,8 +435,7 @@ class Info extends StatefulWidget {
 
 class _InfoState extends State<Info> {
   Future<ApplicationDetails> _getApplicationDetails() async {
-    await Onegini.instance.userClient
-        .authenticateDevice(["read", "write", "application-details"]);
+    await Onegini.instance.userClient.authenticateDevice(["application-details"]);
```

Comment:
if `read` and `write` is needed, you can workaround the issue with
 `.authenticateDevice(["read write application-details"])`

**(but pls double check if that workaround works)**

The fix for that is being addressed

Reply:
This workaround works but read/write not required here in example app to show application-details. So didn't add back.